### ISSUE TITLE
Make it easier to get started with SlicerSALT

### ISSUE THREAD

**Issue by lassoan** [Kitware/SlicerSALT#119]
I've downloaded SlicerSALT because I was interested in what it can do.

I've found "Workflow quick-reference" very useful but I could not easily try these features in a few minutes, so I gave up. I was interested, but right now I was not that much interested so that I spend tens of minutes with trying to find data sets in a suitable format, figure out how exactly modules should be used, etc.

What would have probably changed my experience:
- Have a link from "Workflow quick-reference" to a documentation/tutorial page that contains step by step tutorials that I can follow.
- Have sample data sets and analysis results available by maximum 2-3 clicks (e.g., have a link from "Workflow quick-reference" to jump to SampleData module, and make available relevant sample data sets there)

Note that you can add custom data sets to SampleData module (see example [here](https://github.com/SlicerRt/Sequences/tree/master/SequenceSampleData). If you don't want to show standard Slicer sample data sets then we can add API in Slicer core to remove them.

**Comment by bpaniagua** [comment]
Hi @lassoan Thank you so much for your feedback.
You are right that interfacing data and tutorials need to be streamlined better and easier.
The tutorials are here: http://salt.slicer.org/documentation/ but the data so far is not anywhere within the SALT suite. I think your point is really good, we will work on adding sample datasets.

I will let you know when this is done on this issue we entice you to try the program again :)

Thanks again.

**Comment by lassoan** [comment]
Thanks! I've completely missed the tutorials on http://salt.slicer.org/documentation/ page. I simply did not find them because "General Slicer Guide", "SlicerSALT Forum", and "For Developers" had big green buttons but "Tutorials for the finer steps" was the last one and contained just links that looked like plain text. Maybe "Tutorials" could be moved to be the first column and the links could be made to stand out more (or links in other columns could be made a bit less prominent).

**Comment by bpaniagua** [comment]
Yes, I think you are right! Perception is such a tricky thing, we did not realize that since we were the ones putting the website together!

**Comment by lopezmt** [comment]
To update the subject, we started registering some datasets to test SlicerSALT modules. Until now, we have sample data for the Data Importer, the SPHARM-PDM and the Shape regression module. 

![Sample_data_screen_shot](https://user-images.githubusercontent.com/40871321/59935078-f8752c80-941a-11e9-8c13-1b0fd881b12b.png)

For now, they are available in the Sample Data module but they should be available soon directly from their respective modules
